### ISSUE TITLE
Refactor remove line break between functions

### ISSUE THREAD

**Issue by marcoaaguiar** [sourcery-ai/sourcery#86]
### Issue description or question
I'm running Sourcery via LSP on NeoVim, and when I refactor some code I have the last line break removed.

This code exemplifies why it is annoying:
```python
class Test:
    def f1(self):
        a = 1
        return a

    def f2(self):
        return 2
```
refactors to 
```python 
class Test:
    def f1(self):
        return 1
    def f2(self):
        return 2
```

It probably occurs because Linux/Vim expect a line break ('\n') end of line.
When the edit command comes via LSP it removes the lines (with the linebreak) but does not put a new linebreak in place.
```json
{...
        "edit": {
            "changes": {
                "file:///.../test2.py": [
                    {
                        "range": {
                            "start": {
                                "line": 1,
                                "character": 0
                            },
                            "end": {
                                "line": 3,
                                "character": 300
                            }
                        },
                        "newText": "    def f1(self):\n        return 1"
                    }
                ]
            }
        },
}
```

It is not huge since I'm using format on save.

I'm not sure if it this is fixable, or if there is any option on the LSP to fix this. 
But just want to make you guys know about it.

### Sourcery Version
0.8.0

### Code editor or IDE name and version
NeoVim 0.5.0 (Ubuntu 20.04 via WSL), it does not happen on VSCode (Windows 10)

### OS name and version
Ubuntu 20.04 via WSL (Windows 10)

**Comment by brendanator** [comment]
Thanks for the report. I'll look into this

**Comment by brendanator** [comment]
Fix identified. Will be in the next release

**Comment by Hellebore** [comment]
Fixed with release of 0.8.2.
